src/features/instance/config/overview/index.tsx

```diff
@@ -1,17 +1,31 @@
-import { useState } from 'react';
+import { ReactNode, useState } from 'react';
 import { useParams } from '@tanstack/react-router';
 import { toast } from 'sonner';
 import { getRegistrationInfoQueryOptions } from '@/features/instance/operations/queries/getRegistrationInfo';
 import { useSuspenseQuery } from '@tanstack/react-query';
 import { getConfigurationQueryOptions } from '@/features/instance/operations/queries/getConfiguration';
 import Editor from '@monaco-editor/react';
 import { Button } from '@/components/ui/button';
-import { RemoveInstanceModal } from '../../modals/RemoveInstanceModal';
+import { RemoveInstanceModal } from '@/features/instance/modals/RemoveInstanceModal.tsx';
 import { TextLoadingSkeleton } from '@/components/TextLoadingSkeleton';
 import { useDeleteInstance } from '@/features/cluster/hooks/useDeleteInstance';
 import { getInstanceInfoQueryOptions } from '@/features/instance/operations/queries/getInstanceInfoQuery';
 import { useHumanFileSize } from '@/hooks/useHumanFileSize';
-import { useUpdateRestartInstance } from '../../operations/mutations/updateRestartInstance';
+import { useUpdateRestartInstance } from '@/features/instance/operations/mutations/updateRestartInstance.ts';
+import { isLocalStudio } from '@/config/constants';
```

Comment:
A few extension snuck in to the imports :)

Reply:
Fixed!